inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/extract/MySqlExtractNode.java

```diff
@@ -311,6 +311,9 @@ public String getMetadataKey(MetaField metaField) {
             case UPDATE_BEFORE:
                 metadataKey = "meta.update_before";
                 break;
+            case AUDIT_DATA_TIME:
+                metadataKey = "meta.ts";
+                break;
```

Comment:
Same with case TS.

Reply:
1、to keep consistent with mysql-cdc-flink1.13 and other metadata like meta.update_before
2、to differ from the original ververica implementation of metadata since we may provide different meanings for a meta data in inlong.

Reply:
```java
     case TS:
                metadataKey = "meta.ts";
                break;
```
`CASE AUDIT_DATA_TIME` can be merged with `CASE TS`